packages/addons/addon-depends/system-tools-depends/i2c-tools/package.mk

```diff
@@ -27,6 +27,7 @@ make_target() {
 
 makeinstall_target() {
   make  DESTDIR=${INSTALL} \
+        PREFIX="/usr" \
         prefix="/usr" \
```

Comment:
`prefix="/usr" \` still needed ? I would bet `PREFIX="/usr" \`is enough

Reply:
I’ll double check - but py-smbus is using the little version.
py-smbus/Module.mk:       $(DISTUTILS) install --prefix=$(DESTDIR)$(prefix)

Reply:
https://github.com/buildroot/buildroot/blob/master/package/i2c-tools/i2c-tools.mk
doesn't need it at a quick look

Reply:
Did the validation => needed :-( ... I’ll raise a PR on upstream as a to-fix.

difference between buildroot and us is EXTRA="py-smbus"

see the usr2 and subsequent output.
```
makeinstall_target() {
  make  DESTDIR=${INSTALL} \
        PREFIX="/usr" \
        prefix="/usr2" \
        EXTRA="py-smbus" \
        PYTHON=${TOOLCHAIN}/bin/python3 \
        install
}
```

```
ls -la /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/install_pkg/i2c-tools-4.2/usr2/lib/python3.8/site-packages
total 32
drwxrwxr-x 2 rudi rudi  4096 Jan 13 11:13 .
drwxrwxr-x 3 rudi rudi  4096 Jan 13 11:13 ..
-rw-rw-r-- 1 rudi rudi   288 Jan 13 11:13 smbus-1.1-py3.8.egg-info
-rwxrwxr-x 1 rudi rudi 19848 Jan 13 11:13 smbus.so

ls -la /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/install_pkg/i2c-tools-4.2/usr/lib
total 28
drwxr-xr-x 2 rudi rudi  4096 Jan 13 11:13 .
drwxr-xr-x 6 rudi rudi  4096 Jan 13 11:13 ..
-rw-r--r-- 1 rudi rudi  5840 Jan 13 11:13 libi2c.a
lrwxrwxrwx 1 rudi rudi    11 Jan 13 11:13 libi2c.so -> libi2c.so.0
lrwxrwxrwx 1 rudi rudi    15 Jan 13 11:13 libi2c.so.0 -> libi2c.so.0.1.1
-rwxr-xr-x 1 rudi rudi 10184 Jan 13 11:13 libi2c.so.0.1.1
```

Reply:
My mistake .,,, need to fix the patch.